### ISSUE TITLE
Adaptation to xComfort Bridge?

### ISSUE THREAD

**Issue by oywino** [plamish/xcomfort#31]
Hi, any chance you could adapt/modify this integration to support the xComfort Bridge?

**Comment by BirgerKo** [comment]
Yes that would be good. A person I was in contact with in Eaton stated that
the SHC (Home Controller) is nearing end of life and that the xComfort
Bridge is taking over. Then with Matter support.

man. 19. des. 2022 kl. 22:34 skrev oywino ***@***.***>:

**Comment by plamish** [comment]
Sorry, no plans at my side to work on xComfort Bridge integration. I found there is one existing already: https://github.com/jankrib/ha-xcomfort-bridge. Hope it will work for you

**Comment by oywino** [comment]
That's too bad. We need clever people who can solve this puzzle. I'm currently using @jankrib's solution, but he too seems to be too busy elsewhere to maintain his fine work. Currently it lacks support for a number of important entites, but what he has done is userfriendly and stable.
